apps/src/blockly/customBlocks/googleBlockly/mutators/behaviorDefMutator.js

```diff
@@ -10,6 +10,10 @@
  */
 
 import {ObservableParameterModel} from '@blockly/block-shareable-procedures';
+import {
+  handleLoadDescription,
+  handleSaveDescription,
+} from './functionMutatorHelpers';
```

Comment:
nit: Since (I think) the only files in the `mutators` directory refer to procedure and behavior definitions, could the path be `apps/src/blockly/customBlocks/googleBlockly/mutators/helpers.js` without loss of clarity?

Reply:
I actually had it as `helpers` originally, but I figured we would want to leave open the possibility of other mutators in the future, and then these functions would be somewhat confusing.